sources/packages/web/src/views/institution/locations/programs/LocationProgramAddEdit.vue

```diff
@@ -11,7 +11,7 @@
       <banner
         v-if="programData.hasOfferings"
         :type="BannerTypes.Success"
-        header="Students have applied financial aid for this program"
+        header="Program details no longer editable"
```

Comment:
Is it supposed to be Camelcase? The PR description has it Camelcase.

Reply:
The ticket itself is "Pascal case" but it is not what the UI follows.
@guru-aot can we please have the ticket updated to represent the way that you used it here?
@JasonCTang FYI